alibi/utils/kernel.py

```diff
@@ -0,0 +1,131 @@
+import numpy as np
+from typing import Union, Optional
+from alibi.utils.distance import squared_pairwise_distance
+
+
+class GaussianRBF:
+    def __init__(self, sigma: Optional[Union[float, np.ndarray]] = None, **kwargs) -> None:
+        """
+        Gaussian RBF kernel: :math:`k(x,y) = \\exp(-\\frac{||x-y||^2}{2\\sigma^2})`.
+        A forward pass takes a batch of instances `x` of size `Nx x features` and `y` of size `Ny x features`
+        and returns the kernel matrix of size `Nx x Ny`.
+
+        Parameters
+        ----------
+        sigma
+            Kernel bandwidth. Need not to be specified if being inferred or trained.
+            Can pass multiple values to eval kernel with and then average.
+        **kwargs
+            Other arguments. Not used.
+        """
+        super().__init__()
+        self.config = {'sigma': sigma}
+        self.min_val, self.max_val = 0., 1.   # minimum and maximum values that the kernel can take.
```

Comment:
Are these used anywhere (apart from the `GaussianRBFDistance`, see below)? If not, not sure I see a reason for these?